NetworkConfiguration.md

```diff
@@ -7,44 +7,4 @@ layout: default
 
 # Network Configuration
 
-**DENT offers a wide range of Network Configuration options. These include the following:**
```

Comment:
why did we delete all these sections? DO they exist somewhere else?

Reply:
Before- ![image](https://github.com/dentproject/docs.dent.dev/assets/97520428/3b49be03-d691-43d4-a994-f43b00c9a884)
Now- ![image](https://github.com/dentproject/docs.dent.dev/assets/97520428/92ea8d73-8f0d-4061-94bb-0490538dcd9d)